Lombiq.HelpfulLibraries.LinqToDb/ITableAccessor.cs

```diff
@@ -14,4 +14,13 @@ public interface ITableAccessor
     /// <returns>Linq2db table for query building purposes.</returns>
     ITable<T> GetTable<T>()
         where T : class;
+
+    /// <summary>
+    /// Returns a table for query building purposes.
+    /// </summary>
+    /// <typeparam name="T">Type of the object representing the table in the database.</typeparam>
+    /// <param name="collectionName">Name of the collection.</param>
```

Comment:
Elaborate a bit what a collection is.

Reply:
Based on [this](https://github.com/sebastienros/yessql/blob/0b8dd422407b5216882d2dc990d74ab29c4e56cd/src/YesSql.Abstractions/ISession.cs#L22) comment:
`The name of the collection to store the object in.`

What do you mean?

Reply:
Based on [this](https://github.com/sebastienros/yessql/blob/0b8dd422407b5216882d2dc990d74ab29c4e56cd/src/YesSql.Abstractions/ISession.cs#L22) comment:
`The name of the collection to store the object in.`

What do you mean?

Reply:
"Collection" is a very generic term. Somebody seeing this parameter and the comment might not think of YesSql collections right away.